src/Interactive/HostProcess/x86/.editorconfig

```
@@ -0,0 +1,2 @@
+# Explicitly empty global editorconfig file to ensure we aren't accidentally applying unexpected rules to interactive documents
+global = true
```

I think you meant one of these:

```suggestion
root = true
```

```suggestion
is_global = true
```

---

This file was just moved, not created.